controller/support_bundle_controller.go

```diff
@@ -615,6 +616,12 @@ func (c *SupportBundleController) createSupportBundleManagerDeployment(supportBu
 func (c *SupportBundleController) newSupportBundleManager(supportBundle *longhorn.SupportBundle,
 	nodeSelector map[string]string, tolerations []corev1.Toleration,
 	imagePullPolicy corev1.PullPolicy, priorityClass, registrySecret string) *appsv1.Deployment {
+
+	tolerationSetting, err := c.ds.GetSetting(types.SettingNameTaintToleration)
```

Comment:
I mean you can use `tolerations []corev1.Toleration` instead of querying the setting again.

Reply:
Got it, I was confused because `toleration []corev1.Toleration` is unmarshalled from the setting, directly using it needs to be put back to a string again.

I've moved the unmarshalling into the same method so it won't be called again. PTAL, thank you.